provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/location/LocationTransformer.java

```diff
@@ -1,3 +1,74 @@
 package gov.va.api.health.providerdirectory.service.controller.location;
 
-public class LocationTransformer {}
+import static gov.va.api.health.providerdirectory.service.controller.Transformers.allBlank;
+import static gov.va.api.health.providerdirectory.service.controller.Transformers.convert;
+
+import gov.va.api.health.providerdirectory.service.LocationWrapper;
+import gov.va.api.health.providerdirectory.service.controller.EnumSearcher;
+import gov.va.api.health.stu3.api.datatypes.ContactPoint;
+import gov.va.api.health.stu3.api.resources.Location;
+import gov.va.api.health.stu3.api.resources.Location.LocationAddress;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class LocationTransformer implements LocationController.Transformer {
+
+  LocationAddress address(LocationWrapper.Value value) {
+    return convert(
+        value,
+        ppms ->
+            LocationAddress.builder()
+                .city(ppms.careSiteAddressCity())
+                .line(Collections.singletonList(ppms.careSiteAddressStreet()))
+                .text(ppms.careSiteLocationAddress())
+                .state(ppms.careSiteAddressState())
+                .postalCode(ppms.careSiteAddressZipCode())
+                .build());
+  }
+
+  List<LocationAddress> addresses(LocationWrapper.Value value) {
+    if (value.careSiteAddressCity() == null) {
+      return null;
+    }
+    List<LocationAddress> addressList = new ArrayList<>();
+    addressList.add(address(value));
+    return addressList;
+  }
+
+  @Override
+  public Location apply(LocationWrapper ppmsData) {
+    return location(ppmsData);
```

Comment:
I know many of the data-query transformers are set up like this, but go ahead and inline the `location` method here... the extra layer of indirection doesn't buy us anything.

Reply:
addressed on current branch. (will be pushed with updated tests shortly)